syntax = "proto3";

message MuMcp23008Store {
    repeated bool state = 1;
}

